src/main/scala/com/codacy/client/bitbucket/v2/service/BuildStatusServices.scala

```diff
@@ -62,4 +67,11 @@ class BuildStatusServices(client: BitbucketClient) {
 
     client.putJson(Request(url, classOf[BuildStatus]), payload)
   }
+
+  private def generateCommitBuildStatusesUrl(owner: String, repository: String, commit: String): String = {
```

Comment:
Codacy found an issue: [generateCommitBuildStatusesUrl should have a smaller scope by using private[this]](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)